server/src/internalClusterTest/java/org/elasticsearch/snapshots/CloneSnapshotIT.java

```diff
@@ -377,6 +378,9 @@ public void testMasterFailoverDuringCloneStep1() throws Exception {
 
         awaitNoMoreRunningOperations(internalCluster().getMasterName());
 
+        // Check if the clone operation worked out by chance as a result of the clone request being retried because of the master failover
+        cloneSucceeded = cloneSucceeded ||
```

Comment:
It's mildly annoying that we have to do this kind of thing again (we had this issue in tests with the snapshot `INIT` step as well). There is a reasonable fix here I think (I'll try it when I can find some time but probably not too soon) but for now I opted to just fix the test.

Reply:
Maybe add a //TODO or a descriptive comment somewhere in the code?

Reply:
Added :) in the relevant spots in 060020a